src/appFunctions.js

```diff
@@ -27,4 +27,21 @@ export function changeStatusCard(id, newStatus, {statuses, dataByStatuses}) {
         dataByStatuses: dataByStatusesNew,
         statuses
     });
+}
+
+export function sortCardsByTitle(cards, ascending) {
+    let cardsSort = cards.sort((a, b) => {
+        let charA = a.title.toUpperCase();
+        let charB = b.title.toUpperCase();
+
+        if ((ascending && (charA < charB)) 
+        || (!ascending && (charA > charB))) {
+            return 1;
+        }
+        else {
+            return -1;
+        }
+    });
```

Comment:
what's going on here? why you don't just compare `charA` and `charB` and return 1 or -1?

Reply:
> what's going on here? why you don't just compare `charA` and `charB` and return 1 or -1?

because I need to sort them in ascending or descending order depending on the flag